### ISSUE TITLE
Network Alert notifications are not stacking

### ISSUE THREAD

**Issue by faxalag** [henrypp/simplewall#973]
When network alert notifications stack up over time, there is no way to dismiss the latest notification to view the older ones. The only way to get to the older notifications is by clicking either "Allow" or "Block" on the latest alert. Clicking the "X" on the top right of the window dismisses all the notifications.

If I remember correctly, there used to be an "X" button beside the "Block" button which dismisses only the latest alert. Please add it back, thanks!

**Comment by ltGuillaume** [comment]
Yeah, I don't understand this either. We had a whole discussion about this some time ago, for example in #84, and all of a suddent it's gone again.
